### DISCUSSION TITLE
SSR support

### DISCUSSION THREAD

**Discussion by galvez** [alpinejs/alpine#782] [4. General]
I couldn't find any mention of "server" in the README so I'm guessing this isn't supported at al? 

Can you confirm @calebporzio?

**Comment by ryangjchandler** [comment]
@galvez Alpine doesn't officially support server-side rendering. The main use case for Alpine is alongside statically rendered HTML or server-rendered content from a templating engine such as Laravel Blade, Nunjucks, Twig, etc..
